src/classes/window/overview.vala

```diff
@@ -390,7 +390,11 @@ namespace pdfpc.Window {
             // draw slide number
             var layout = Pango.cairo_create_layout(cr);
             layout.set_font_description(this.font_description);
-            layout.set_text(@"$(slide_id + 1)", -1);
+            if (this.metadata.get_user_slide_hidden(slide_id)) {
+                layout.set_markup(@"<s>$(slide_id + 1)</s>", -1);
```

Comment:
For me the strike through is not easily recognisable. Is there another way for highlighting these slides? Maybe change the colour of the text or grey out the slide within the overview?

Reply:
Right, it is not obvious. Let me think about it. The options are rather limited, though, if we try to stay within the essentially grayscale pdfpc palette...

Reply:
Is it better now?

Reply:
Yes!